extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/internal/discovery/SoundTouchDiscoveryParticipant.java

```diff
@@ -93,6 +94,12 @@ public DiscoveryResult createResult(ServiceInfo info) {
             if (getMacAddress(info) != null) {
                 properties.put(BoseSoundTouchConfiguration.MAC_ADDRESS, new String(getMacAddress(info)));
             }
+            
+            // Set manufacturer as thing property (if available)
+            byte[] manufacturer = info.getPropertyBytes("MANUFACTURER");
+            if (manufacturer != null) {
+                properties.put(Thing.PROPERTY_VENDOR, new String(manufacturer));

```

Comment:
Just passing by, but I think you should normally not use this constructor because it depends on the platform's default charset, better pick one explicitly. And then of course also adapt the existing occurrences on line 95 and maybe more in this file.